app/models/trn_request.rb

```diff
@@ -1,3 +1,8 @@
 # frozen_string_literal: true
 class TrnRequest < ApplicationRecord
+  def answers_checked=(value)
+    return unless value
+
+    self.checked_at = Time.current
+  end
```

Comment:
Good point. I'll add a unit test for this to document the intended behaviour.

I can see that this would be a useful field to have long term as a record of when this happened but also think we'd add some sort of FSM as well to handle the actual transitions.